src/App.js

```diff
@@ -4,35 +4,54 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+    findMovies: moviesFromServer,
+  };
 
-  render() {
-    return (
-      <div className="page">
-        <div className="page-content">
-          <div className="box">
-            <div className="field">
-              <label htmlFor="search-query" className="label">
-                Search movie
-              </label>
+changeQuery = (event) => {
+  this.setState({
+    query: event.target.value,
+    findMovies: moviesFromServer.filter(
```

Comment:
You always use the same array - `moviesFromServer` (without any changes) so you may filter it in method `render` directly without writing it to the state. Just create variable `findMovies`  in 'render', filter there `MoviesFromServer` and pass it to the `MoviesList`